src/resolvers/findById.js

```diff
@@ -30,8 +31,9 @@ export default function findById<TSource: MongooseDocument, TContext>(
 
       if (args._id) {
         resolveParams.query = model.findById(args._id); // eslint-disable-line
+        resolveParams.model = model; // eslint-disable-line
         projectionHelper(resolveParams);
-        return resolveParams.query.exec();
+        return beforeQueryHelper(resolveParams, model);
```

Comment:
Forgot to remove second arg (model)?
All resolvers affected.

Reply:
nice catch :)